pkg/arm/v5/data/master-startup.sh

```diff
@@ -12,6 +14,7 @@ if ! grep /var/lib/docker /etc/fstab; then
 }
 EOF
   systemctl start docker.service
+  systemctl start docker-cleanup.timer
```

Comment:
do you not have to restart systemctl stop docker-cleanup.service as well here?

Reply:
I believe starting the timer will start the service, but I had the same question.

Reply:
the service is oneshot.  The timer starts it each time it goes off.

Reply:
Ok now I get it, thanks.